### PR TITLE
[plugwiseha] Add 5 boiler related channels

### PR DESCRIPTION
**Pull request by lsiepel**
Adds five channels:

- burnerStartsFailed
- burnerStarts
- burnerIgnitionsFailed
- burnerOpTime
- burnerDHWOpTime

Other:
- While looking at the state patterns, the percentage channels where also improved.

### CODE REVIEW

**Comment by lolodomo** [comment]
You will have to resolve a file conflict after I merged your other PR.

File: bundles/org.openhab.binding.plugwiseha/src/main/resources/OH-INF/thing/channels.xml [hunk 1]
```diff
@@ -120,6 +120,49 @@
...
+			<tag>Status</tag>
+			<tag>Duration</tag>
+		</tags>
+		<state pattern="%.0f %unit%" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="burnerDHWOpTime">
+		<item-type unitHint="h">Number:Time</item-type>
+		<label>DHW Operation Time</label>
+		<description>Total operation time for domostic hot water</description>
```
Comment (lolodomo) [inline bundles/org.openhab.binding.plugwiseha/src/main/resources/OH-INF/thing/channels.xml:158]: typo: domestic ?

File: bundles/org.openhab.binding.plugwiseha/src/main/resources/OH-INF/thing/channels.xml [hunk 2]
```diff
@@ -145,11 +188,11 @@
 	</channel-type>
 
 	<channel-type id="modulationLevel">
-		<item-type>Number</item-type>
+		<item-type>Number:DimensionLess</item-type>
```
Comment (lolodomo) [inline bundles/org.openhab.binding.plugwiseha/src/main/resources/OH-INF/thing/channels.xml:191]: The state built for this channel is already a QuantityType ?
I saw no change in that PR for this channel.
What about thing migration ? This leads to the discussion we have for color temperature channels. If we change the item type, I guess that the channel must be removed and added through the migration tool. But it has to be verified.

File: bundles/org.openhab.binding.plugwiseha/src/main/resources/OH-INF/thing/channels.xml [hunk 3]
```diff
@@ -249,15 +292,15 @@
 	</channel-type>
 
 	<channel-type id="valvePosition">
-		<item-type>Number</item-type>
+		<item-type>Number:DimensionLess</item-type>
```
Comment (lolodomo) [inline bundles/org.openhab.binding.plugwiseha/src/main/resources/OH-INF/thing/channels.xml:295]: Same here

**Review by lolodomo — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.plugwiseha/src/main/resources/OH-INF/thing/channels.xml (hunk 2 above)
Comment (lolodomo) [reply bundles/org.openhab.binding.plugwiseha/src/main/resources/OH-INF/thing/channels.xml:191]: And if you change type, README has to be updated too.

File: bundles/org.openhab.binding.plugwiseha/src/main/resources/OH-INF/thing/channels.xml (hunk 1 above)
Comment (lsiepel) [reply bundles/org.openhab.binding.plugwiseha/src/main/resources/OH-INF/thing/channels.xml]: Nice catch. Thanks.

File: bundles/org.openhab.binding.plugwiseha/src/main/resources/OH-INF/thing/channels.xml (hunk 2 above)
Comment (lsiepel) [reply bundles/org.openhab.binding.plugwiseha/src/main/resources/OH-INF/thing/channels.xml:191]: Fixed readme. 
About the migration i'm not sure. Does it also apply to dimensionless? I use file based configuration. But obviously i can test this.

Comment (lsiepel) [reply bundles/org.openhab.binding.plugwiseha/src/main/resources/OH-INF/thing/channels.xml:191]: Tested and added upgrade instructions as indeed they are needed.
Minor difference that i performed an update and not add + remove. Checked the jsondb and the item type is updated.

File: bundles/org.openhab.binding.plugwiseha/src/main/resources/OH-INF/thing/channels.xml (hunk 3 above)
Comment (lsiepel) [reply bundles/org.openhab.binding.plugwiseha/src/main/resources/OH-INF/thing/channels.xml:295]: See other comment. Fixed.

File: bundles/org.openhab.binding.plugwiseha/src/main/resources/OH-INF/thing/channels.xml (hunk 2 above)
Comment (lolodomo) [reply bundles/org.openhab.binding.plugwiseha/src/main/resources/OH-INF/thing/channels.xml:191]: Fine

**Review by lolodomo — APPROVED** [review]
LGTM, thank you

**Pull request merged**